### PR TITLE
[OSDOCS-4882]:HCP cluster AWS managed policies

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.13+
Issue:

https://issues.redhat.com/browse/OSDOCS-4882
Link to docs preview:

https://59437--docspreview.netlify.app/openshift-rosa/latest/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html#rosa-sts-creating-account-wide-sts-roles-and-policies_rosa-hcp-sts-creating-a-cluster-quickly
QE review:
- [x] QE has approved this change.

Additional information:

image

### CODE REVIEW

**Comment by mletalie** [comment]
image

**Comment by mletalie** [comment]
Hello @yuwang-RH, when you get a moment can you please provide QE review? Thanks!

**Comment by xueli181114** [comment]
@siserafin Would you please help to review the doc?

**Comment by siserafin** [comment]
> @siserafin Would you please help to review the doc?

sure

**Comment by siserafin** [comment]
\lgtm

**Comment by mletalie** [comment]
Hello @arendej,
    As an FYI, although we had discussed removing the "Sample Output" section of the affected docs b/c the image did not correctly reflect the output of account roles for an HCP cluster, I was able to edit it so it would be presented accurately. If you feel like having the output shown at all is not beneficial to the reader, I can remove it. Otherwise I will keep the section in the docs. 
image

**Comment by arendej** [comment]
> Hello @arendej, As an FYI, although we had discussed removing the "Sample Output" section of the affected docs b/c the image did not correctly reflect the output of account roles for an HCP cluster, I was able to edit it so it would be presented accurately. If you feel like having the output shown at all is not beneficial to the reader, I can remove it. Otherwise I will keep the section in the docs. 

I still think the sample output should be removed because its content is not accurate, and it deviates from how we presented this section in the Classic workflow.

**Comment by mletalie** [comment]
Hi @arendej,
    I have removed the sample output as per your request. Time permitting, I would like to know what the proper output should be for HCP list account roles, as I thought what I had presented was correct. Thanks!

**Comment by bmcelvee** [comment]
LGTM! Waiting for the go-ahead to merge :)

File: modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc
Comment (oriAdler) [inline modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc:25]: We don't need the `force` flag, as the policies are managed by AWS and already exist in each AWS account.
```suggestion
$ rosa create account-roles --hosted-cp
```

File: modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc
Comment (oriAdler) [inline modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc:32]: We haven't yet enforced managed policies for HCP, but my understanding is that we will enforce managed policies in GA.

File: modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc
Comment (oriAdler) [inline modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc:16]: We have enabled creating HCP account roles with managed policies in release `1.2.26`, before that it was blocked in the production environment.
```suggestion
* You have installed and configured ROSA CLI (`rosa`) version 1.2.26 or later on your installation host.
```

File: modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc
Comment (oriAdler) [inline modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc:21]: NP
Since the policies already exist in each AWS account and are managed by AWS, we are just attaching them to the roles, but not creating the policies.
```suggestion
. If they do not exist in your AWS account, create the required account-wide STS roles and attach the policies by running the following command:
```

File: modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc
Comment (oriAdler) [inline modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc:28]: We don't use this flag in combination with the `--hosted-cp` flag, as the policies are updated by AWS, and not created by the user or the ROSA CLI, we are only attaching existing policies.

File: modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc
Comment (oriAdler) [reply modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc:32]: Please ignore my recent comment, as we are going to enforce managed policies soon in GA, this paragraph is accurate.

**Review by wgordon17 — APPROVED** [review]
LGTM

**Pull request merged**
